### PR TITLE
rename pypi package to pywithings

### PR DESCRIPTION
**Pull request by brad**
@orcasgit/orcas-developers Please review. This changes the name of the package so we can publish our own version on PyPi

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2012 Maxime Bouroumeau-Fuseau
+Copyright (C) 2012-2017 Maxime Bouroumeau-Fuseau, and ORCAS
```
Comment (grokcode) [inline LICENSE]: @brad from what I found on stackoverflow, we want to say
```
Original work Copyright (C) 2012 Maxime Bouroumeau-Fuseau
Modified work Copyright (C) 2017 ORCAS
```

https://softwareengineering.stackexchange.com/questions/157968/how-to-manage-a-copyright-notice-in-an-open-source-project

File: setup.py [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
 
 setup(
-    name='withings',
+    name='pywithings',
     version='0.4.0',
     description="Library for the Withings API",
-    author='Maxime Bouroumeau-Fuseau',
-    author_email='maxime.bouroumeau@gmail.com',
-    url="https://github.com/maximebf/python-withings",
+    author='Maxime Bouroumeau-Fuseau, and ORCAS',
```
Comment (grokcode) [inline setup.py]: @brad I think we can remove Maxime Bouroumeau-Fuseau here so that people don't try to contact him at developer@orcasinc.com

Comment (brad) [inline setup.py]: @grokcode Oh good point :+1:

File: LICENSE (hunk 1 above)
Comment (brad) [inline LICENSE]: @grokcode Cool, thanks! I didn't know about that

**Comment by grokcode** [comment]
@brad Can we also put a note in the readme that this was forked with a link back to https://github.com/maximebf/python-withings?

**Comment by grokcode** [comment]
LGTM! :shipit:

**Pull request merged**